INSTALL.md

```diff
@@ -45,29 +45,29 @@ On redhat based systems (like Fedora, RHEL), it'll be `elfutils-devel`.
 
     $ sudo dnf install elfutils-devel
 
-It also uses libstdc++ library to demangle C++ symbols in full detail.
+The libdw library also uses libstdc++ library to demangle C++ symbols in full detail.
```

Comment:
The context might be a bit confusing but `It` means `uftrace` here. So I would suggest this.
```suggestion
uftrace also uses libstdc++ library to demangle C++ symbols in full detail.
```